__tests__/nc_news_test.test.js

```diff
@@ -356,4 +356,29 @@ describe('GET /api/topics', () => {
                 expect(response.body.msg).toBe("Bad Request")
             })
         })
-    });
\ No newline at end of file
+    });
+    describe('GET /api/users', () => {
+        test('GET /api/users responds with an array of objects, each possessing username, name and avatar_url properties', () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then((response)=> {
+                const users = response.body.users
+                
+                expect(users.length).toBe(4)
+                users.forEach((user)=>{
+                    expect(user).toHaveProperty("username");
+                    expect(user).toHaveProperty("name");
+                    expect(user).toHaveProperty("avatar_url");
+                })
+            })
+        })
+        test('GET /api/users responds with error 404a and \'Path Not Found\', when requested with a spelling mistake', () => {
+            return request(app)
+            .get("/api/userssss")
+            .expect(404)
+            .then((response)=>{
+                expect(response.body.msg).toBe("Path Not Found")
```

Comment:
This error is not specific to this endpoint, a PATH-NOT-FOUND style error like this only needs to be tested once in the whole API.
This is because no matter how many different ways you misspell an endpoint, the API will still handle it in exactly the same way 👍